### DISCUSSION TITLE
SCENIC+ on bulk RNA+ATAC data

### DISCUSSION THREAD

**Discussion by itraniello** [aertslab/scenicplus#362] [General]
Hello,

Thank you for developing this wonderful tool! 

I have paired, bulk brain RNA+ATAC-seq data generated in a non-model insect species. I'm wondering if SCENIC+, like pySCENIC, could be applied to bulk data, or if I'm missing something obvious in the pipeline that would preclude this possibility. If there is an untested possibility of this working, I would be more than happy to try the pipeline and share findings, but some indication of feasibility would be good before I get started =).

Thank you!

Ian

**Comment by SeppeDeWinter** [comment]
Hi Ian

Thanks for the nice words.

In principle it is possible to use it on bulk data. 
I suppose you have multiple samples?

I would recommend to generate fake single cells by sampling a set number of reads from each bam file.

Best,

Seppe

**Reply by itraniello** [reply]
Hi Seppe,

Thanks for the rapid response! 

Yes, we have > 100 samples, all paired RNA+ATAC from the whole brain (with lots of cool behavioral/developmental comparisons to be made). 

To make sure I understand your suggestion: for each sample, this would entail using the RNA- and ATAC-Seq BAM files to simulate single-cells, running SCENIC+ on these fake "cell types," then consolidating results at the end with the understanding that true signal is more likely to reoccur across each of the simulated cell types (since we're starting with bulk data). Something like that, or am I totally wrong?

Thanks again,

Ian

**Reply by SeppeDeWinter** [reply]
Wow, > 100 samples, that's cool!
Yes that's indeed what I meant with generating fake single cells. However, given that you have so many samples you could also try without generating fake single cells. 

Best,

Seppe

**Reply by itraniello** [reply]
Great! Thank you so much for the feedback.

**Reply by SeppeDeWinter** [reply]
No worries, good luck with the analysis!

**Comment by j-andrews7** [comment]
@itraniello did you ever give this a shot? Interested in hearing if it was worthwhile before I embark on a similar path.

**Reply by itraniello** [reply]
Yes, @j-andrews7! And everything ran quite smoothly. I was comparing several groups/treatments for a behavioral analysis, so I essentially treated each sample (one "bulk" RNA/ATAC-Sequenced brain) as a "cell" and the group/treatment as an annotated "cell type."
I would be happy to follow up, feel free to DM me (it4770@princeton.edu).

**Reply by lxinghua** [reply]
Hi,

How is this analysis going? I share a very similar situation. I have 70 samples, separated bulk RNA+ATAC-seq data. Could you share the pipeline with me (lxinghua@umich.edu). I am not quite familiar with the single cell analysis. I'm struggling in making R objects from the bulk analysis matrix. Could you introduce briefly the major steps?

Thanks,
Xinghua

**Reply by itraniello** [reply]
Hi Xinghua -- I will email you since the discussion might run longer than appropriate for this forum =).
But @SeppeDeWinter there seems to be some increasing interest in applying SCENIC+ to bulk data, and I'd be happy to contribute my workflow for discussion.

**Reply by aine-mccolgan** [reply]
Hello, could I please ask about this as well? (aine.mccolgan@crick.ac.uk) I'm trying to work out whether it would be appropriate to use bulk data for the analysis I want to do before resorting to generating my own single-cell data!
